build.sbt

```diff
@@ -110,9 +110,29 @@ val monix = crossProject(JVMPlatform, JSPlatform)
 val monixJVM = monix.jvm
 val monixJS  = monix.js
 
+val zio = crossProject(JVMPlatform, JSPlatform)
+  .in(file("modules/zio"))
+  .jvmConfigure(_.dependsOn(coreJVM))
+  .settings(moduleSettings)
+  .settings(
+    resolvers += Resolver.sonatypeRepo("releases"),
+    libraryDependencies ++= Seq(
+      "dev.zio"       %%% "zio"         % "1.0.0-RC11-1",
```

Comment:
How long do you think it will be before we see ZIO 1.0.0? Is it worth waiting for that? As I understand it, cats-effect 2.0.0 and Monix 3.0.0 should be out very soon, so it would be nice to do a release with all the dependencies on stable versions.

Reply:
I don't know when it will be release but I think it will be soon. I agree we should wait stable versions to make releases.